demos/petclinic/readme.md

```diff
@@ -0,0 +1,11 @@
+# Spring PetClinic
+
+This docker file contains [Spring PetClinic](https://github.com/spring-projects/spring-petclinic) sample built on top of chiselled-jre container.
```

Comment:
```suggestion
This Dockerfile runs the [Spring PetClinic](https://github.com/spring-projects/spring-petclinic) sample on the `ubuntu/chiselled-jre` container image.
```

Reply:
I've applied those manually, github probably did not like renamed file =)